### ISSUE TITLE
Unable to use EAP-SIM authentication method for Wi-Fi networks

### ISSUE THREAD

**Issue by y0lopix** [DeadSquirrel01/lineage-16_a5_releases#15]
The SIM authentication method isn't available for networks secured by EAP protocol. Also the settings app crashes when clicking on a known Wi-Fi network already authenticated with the SIM card. Here is the logcat: [logcat.txt](https://github.com/DeadSquirrel01/lineage-16_a5_releases/files/2630186/logcat.txt)

**Comment by DeadSquirrel01** [comment]
SIM-EAP is broken on 15.1 and 16.0 in lineageos side. I think the implementation of eap proxy doesn't work anymore in newer android versions.
